app/scenes/Settings/People.js

```diff
@@ -40,6 +40,10 @@ class People extends React.Component<Props> {
     this.inviteModalOpen = false;
   };
 
+  fetchPage = params => {
+    return this.props.users.fetchPage({ ...params, includeSuspended: true });
```

Comment:
🍖  Because suspended users are no longer included by default we need to explicitly request them on the people management screen.

Note to self: This data loading is due a revamp.